onset_maproom/maproom.py

```diff
@@ -82,6 +84,52 @@ def map_click(click_lat_lng):
         children=dlf.Tooltip("({:.3f}, {:.3f})".format(*lat_lng))
     )
 
+@APP.callback(
+    Output("plotly_onset_test", "figure"),
+    Input("map", "click_lat_lng"),
+    Input("earlyStartDay", "value"),
+    Input("earlyStartMonth", "value"),
+    Input("searchDays", "value"),
+    Input("wetThreshold", "value"),
+    Input("runningDays", "value"),
+    Input("runningTotal", "value"),
+    Input("minRainyDays", "value"),
+    Input("dryDays", "value"),
+    Input("drySpell", "value"),
+)
+def onset_plot(click_lat_lng, earlyStartDay, earlyStartMonth, searchDays, wetThreshold, runningDays, runningTotal, minRainyDays, dryDays,drySpell):
+    lat, lng = get_coords(click_lat_lng)
+    params = {
+        "earlyStart": str(earlyStartDay) + " " + earlyStartMonth,
+        "searchDays": searchDays,
+        "wetThreshold": wetThreshold,
+        "runningDays": runningDays,
+        "runningTotal": runningTotal,
+        "minRainyDays": minRainyDays,
+        "dryDays": dryDays,
+        "drySpell": drySpell
+    }
+    ds = rr_mrg.sel(X=lng, Y=lat, method="nearest")
+    onset = calc.onset_date(ds.precip, int(earlyStartDay),\
+        calc.strftimeb2int(earlyStartMonth), params["searchDays"],\
+        params["wetThreshold"], params["runningDays"], params["runningTotal"],\
+        params["minRainyDays"], params["dryDays"], params["drySpell"])
+    #onset = calc.onset_date(ds.precip, int(earlyStartDay,\
+    #    calc.strftimeb2int(earlyStartMonth), \
+    #    searchDays, wetThreshold, runningDays, runningTotal, \
+    #    minRainyDays, dryDays, drySpell)
+    onsetDate = onset["T"] + onset
+    year = pd.DatetimeIndex(onsetDate['T']).year
+    onsetMD = onsetDate.dt.strftime("1999-%m-%d")
+    graph = px.line(x=onsetMD['T'], y=onsetMD)
```

Comment:
Did you mean to use `year` for the x axis? I currently see years like `0.6x10^18".

Reply:
When I run the app this does not happen.. so I am wondering why there are discrepancies in the way the app is running for the two of us? The graph shows the x axis as integers years~ 1981-2020